doc/UsersGuide/source/porting.rst

```diff
@@ -286,6 +286,93 @@ This construct is already accepted by some Modelica tools, but is not yet
 included in the current Modelica specification 3.5, nor even in the current working
 draft of 3.6, so it is not currently supported by OpenModelica.
 
+Incomplete specification of initial conditions
+------
+The simulation of Modelica models of dynamical systems requires the tool to
+determine a consistent initial solution for the simulation to start. In
+principle, this can be done by adding to the system equations one initial
+condition for each continuous state variable (after index reduction) and one
+initial condition for each discrete variable, then solving the resulting
+initialization problem. These initial conditions can be formulated by adding
+a *start = <expression>* and a *fixed = true* attribute to those variables, e.g.
+
+.. code-block:: modelica
+
+  parameter Real v_start = 2.5;
+  Real x(start = 10, fixed = true);
+  discrete Real v(start = v_start, fixed = true);
+  Integer i(start = 2, fixed = true);
+
+or by adding initial equations, e.g.:
+
+.. code-block:: modelica
+
+    parameter Real x_start = 10;
+    Real x;
+    Real y(start = 3.5);
+    discrete Real v;
+    Integer i;
+  initial equation
+    x = x_start;
+    der(y) = 0;
+    v = 2.5;
+    i = 2;
```

Comment:
Should't this reflect the same initial conditions for `x`, `v`, and `i` as the previous snippet with attributes?

Reply:
Your're right, it could be confusing otherwise. I've just updated it with a new commit.